doc/source/install.rst

```diff
@@ -72,6 +72,285 @@ and start hacking it with:
 This command will provide you a shell with all the dependencies required.
 
 
+Guix
+----
+If you are running `Guix System <https://guix.gnu.org/>`__ or you have the
+`guix <https://guix.gnu.org/>`__ package manager installed, you can install
+papis by running:
+
+::
+
+  guix package --install-from-file=papis.scm
+
+Where papis.scm file defines the papis package recipe (this is a first draft, some failing tests are switched off):
+
+.. code:: scheme
```

Comment:
This whole chunk of scheme code could be a separate file in `contrib/papis.scm` and here just use
```rst
Where :download:`papis.scm <../../contrib/papis.scm>` file defines the papis package recipe.
```
? We don't need to paste the whole code in the documentation, since it's quite long.